_articles/zh-cn/how-to-contribute.md

```diff
@@ -499,9 +499,9 @@ related:
 
 如果你做了所有该做的事情，还是没有人理你，那就是真的没有人对你的贡献做出响应。这可能感觉上不太好受，但是千万不要灰心。每个人都会遇到这样的情况。其实有太多种原因没有人响应你的提交了，包括很多个人情形都是不在你控制范围的。再接再厉，换一种方法去提交，或者换一个项目。这年头，很多社区成员都在积极的参与和响应他人，都在抢夺优秀的人才，若没有人搭理你，你换地方是没有任何不对的地方的。
 
-### 🚧 其他人的要求你对自己的提交做出变更。
+### 🚧 有人要求你对自己的提交做出变更。
 
-对于自己的提交作出变更这件事非常的普遍，可能是你获得了反馈，也可能是变更了代码。
+被要求对自己的提交作出变更是很常见的，无论是对你的想法反馈，还是对你的代码变更上的反馈。
```

Comment:
I think that this changes the meaning of the sentence to something other than its original intention.

Reply:
Ok, I got it wrong, I understand it like this: 

whether that's feedback on the scope of **(your idea, or changes to your code)**.

Reply:
Forgive me (google translate), but does this make sense?
这是很常见的被要求修改您的提交，无论是对你在你的代码更改实现或反馈。

I do think there is an opportunity to improve the original text. Basically I think we want it to say:

> It is very common to be asked to make changes to your submission, whether it is feedback on your implementation or feedback on your code changes.

Reply:
> Forgive me (google translate), but does this make sense?
> 这是很常见的被要求修改您的提交，无论是对你在你的代码更改实现或反馈。

This Chinese translation is a bit strange, but it can express sense.

> It is very common to be asked to make changes to your submission, whether it is feedback on your implementation or feedback on your code changes.

I think it can be translated as:

被要求对你的提交进行更改是很常见的，无论是对你的实现上的反馈，还是你代码改动上的反馈

Reply:
Great! Can you make those changes to the PR and we can get this merged in?

Reply:
@kenyonj Ok, I have modified

Reply:
1. It is common to be asked to make changes to your submission, whether it's feedback on your ideas or feedback on changes to your code.
2. It is very common for you to make changes to your submission, either because you have received feedback or because you may have changed your code.

This is the closest translation so please suggest what to assume in regards to the contribution terms